wireapp/outlook-addin
# Kj/wir 13 user experience enhancements settings conversation creation

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues
- "Disconnect Add-in" instead of "Log Out"
- Name of Conversation should be set to CLDR:: + Meeting picked date or if subject exists CLDR:: {Meeting Name}
- New Design for Settings Section
### Solutions

- Renamed the Button
- modifyConversationName, changes the name of the conversation by given details
-  New Design for Settings Section 
### Testing

#### How to Test

- Check the Button on the "Settings" Page
- Check the /v5/conversations endpoint at the network tab and look at the name attribute
- Open Settings Tab

#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
